### PR TITLE
add ocil to rsyslog_nolisten

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- ocil was missing

#### Rationale:

- stig effort

### CODE REVIEW

File: linux_os/guide/system/logging/rsyslog_accepting_remote_messages/rsyslog_nolisten/rule.yml [hunk 1]
```diff
@@ -41,3 +41,16 @@ references:
     cis-csc: 1,11,12,13,14,15,16,18,3,4,5,6,8,9
     stigid@rhel7: RHEL-07-031010
     cis@rhel8: 4.2.1.6
+
+ocil_clause: "rsyslog accepts remote messages"
+
+ocil: |-
+    Display the contents of the configuration file:
+    <pre>cat /etc/rsyslog.conf</pre>
+    Make sure that following lines are not present in the configuration:
```
Comment (jan-cerny) [inline linux_os/guide/system/logging/rsyslog_accepting_remote_messages/rsyslog_nolisten/rule.yml:50]:
```suggestion
    Make sure that the following lines are not present in the output:
```

**Comment by JAORMX** [comment]
@matejak this seems to be a legitimate issue in building images. I don't know why other builds are working (maybe it's using a chaced one)... anyway. Fixing it here https://github.com/ComplianceAsCode/content/pull/6090

**Comment by matejak** [comment]
test this please

**Comment by matejak** [comment]
test this please

**Comment by jan-cerny** [comment]
@vojtapolasek thanks

**Pull request merged**